jquery.dataTables/jquery.dataTables.d.ts

```diff
@@ -189,6 +189,17 @@ declare namespace DataTables {
         draw(reset?: boolean): DataTable;
 
         /*
+        * Look up a language token that was defined in the DataTables' language initialisation object.
+        *
+        * @param token The language token to lookup from the language object.
+        * @param def The default value to use if the DataTables initialisation has not specified a value.
+        * @param numeric If handling numeric output, the number to be presented should be given in this parameter. If not numeric operator is required (for example button label text) this parameter is not required.
+        *
+        * @returns Resulting internationalised string.
+        */
+        i18n(token: string, def: Object | string, numeric?: number): string;
```

Comment:
I can do that, but to be clear, the documentation defines the 'def' parameter as object or string: https://datatables.net/reference/api/i18n(). What do you want me to do?